pkg/scheduler/event_handler.go

```diff
@@ -256,6 +258,49 @@ func (s *Scheduler) reconcileCluster(key util.QueueKey) error {
 	)
 }
 
+func (s *Scheduler) reconcileClusterResources(key util.QueueKey) error {
+	clusterName, ok := key.(string)
+	if !ok {
+		return fmt.Errorf("invalid cluster key: %s", key)
+	}
+	return utilerrors.NewAggregate([]error{
+		s.enqueueUnschedulableBindings(clusterName),
+		s.enqueueUnschedulableCRBs(clusterName)},
+	)
+}
+
+func (s *Scheduler) enqueueUnschedulableBindings(clusterName string) error {
+	klog.V(4).Infof("Enqueue Unschedulable ResourceBinding with cluster %s", clusterName)
+	bindings, _ := s.clusterBindingLister.List(labels.Everything())
```

Comment:
clusterBindingLister -> bindingLister

Reply:
Will there be performance issues if all the rb in this list are looped and processed because there are too many rb?

Reply:
> Will there be performance issues if all the rb in this list are looped and processed because there are too many rb?


I don't think so, the `cluster` object changes once every 30s, so the call frequency here is very low.